apps/tenant-management-webapp/src/app/pages/admin/services/events/stream/addEditStream/addEditStream.tsx

```diff
@@ -83,175 +84,169 @@ export const AddEditStream = ({ onSave, eventDefinitions, streams }: AddEditStre
 
   return (
     <StreamModalStyles>
-      {isOpen && (
-        <GoAModal
-          testId="stream-form"
-          open={true}
-          heading={isEdit ? 'Edit stream' : 'Add stream'}
-          actions={
-            <GoAButtonGroup alignment="end">
-              <GoAButton
-                testId="form-cancel"
-                type="secondary"
-                onClick={() => {
-                  validators.clear();
-                  dispatch(ResetModalState());
-                  setStream(initialStream);
-                }}
-              >
-                Cancel
-              </GoAButton>
-              <GoAButton
-                type="primary"
-                testId="form-save"
-                disabled={!stream?.name || validators.haveErrors()}
-                onClick={() => {
-                  if (!isEdit && validators['duplicate'].check(stream.id)) {
-                    return;
-                  }
-                  onSave(stream);
-                  dispatch(ResetModalState());
-                }}
-              >
-                Save
-              </GoAButton>
-            </GoAButtonGroup>
-          }
-        >
-          <GoAFormItem error={errors?.['name']} label="Name">
-            <GoAInput
-              type="text"
-              name="stream-name"
-              width="100%"
-              value={stream.name}
-              disabled={isEdit}
-              testId="stream-name"
-              aria-label="stream-name"
-              onChange={(name, value) => {
-                validators.remove('name');
-                validators['name'].check(value);
-                const streamId = toKebabName(value);
-                setStream({ ...stream, name: value, id: streamId });
+      <GoAModal
+        testId="stream-form"
+        open={isOpen}
+        heading={isEdit ? 'Edit stream' : 'Add stream'}
+        actions={
+          <GoAButtonGroup alignment="end">
+            <GoAButton
+              testId="form-cancel"
+              type="secondary"
+              onClick={() => {
+                validators.clear();
+                dispatch(ResetModalState());
+                setStream(initialStream);
+                cleanUpGoAModalLeftStyle();
```

Comment:
You dont need this method "cleanUpGoAModalLeftStyle", removing "isOpen && (" should be good enough

Reply:
Oh, I just updated the code. Thanks, @thdlam